ideascube/library/static/library/library.css

```diff
@@ -0,0 +1,42 @@
+/* ************************************************* */
```

Comment:
I'd also add that if doing multiple requests for CSS becomes a problem, we could just pre-process the CSS to merge them into a big one.

It's often annoying, but there's a reason pretty much everybody does it: it allows you to split your work in a way that makes sense to you (the developer) while still bundling everything in a way that makes sense to users (and servers, since they have to answer less requests).

But since it's annoying to set up, we probably shouldn't deal with that until we find we absolutely have to. :laughing: